lib/godwoken_explorer/graphql/resolovers/transaction.ex

```diff
@@ -31,6 +32,14 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
     end
   end
 
+  def method_id(%{method_id: method_id}, _args, _resolution) do
+    case method_id |> to_string() do
+      "" -> {:ok, nil}
+      "0x00" -> {:ok, nil}
```

Comment:
It looks like a monkey patch? When will `method_id` be `0x00` in the workflow?

Reply:
When the transaction is native transfer, the method_id should be nil not `0x00`